### PR TITLE
readme: add project links

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ A container to create disk images from [bootc](https://github.com/containers/boo
 especially oriented towards [Fedora/CentOS bootc](https://docs.fedoraproject.org/en-US/bootc/) or
 derivatives.
 
+## Project
```
Comment (mvo5) [inline README.md:7]: I'm slightly confused, it seems we already have this in https://github.com/osbuild/bootc-image-builder/blob/main/README.md#-project (i.e. much further down in this file)

**Review by mvo5 — COMMENTED** [review]
Inline question, maybe I'm missing something though :)

File: README.md (hunk 1 above)
Comment (lzap) [reply README.md:7]: Good point, I opened a dozen of these PRs and overlooked this. Deleted the section down, I think it is better at the top this is where all other projects have it.

**Comment by lzap** [comment]
I am not sure how I managed push to upstream repo, I think I had to use github editor. Closing see here: https://github.com/osbuild/bootc-image-builder/pull/717

**Pull request closed without merging**